### PR TITLE
fix: delete error log when file enoent.

### PR DESCRIPTION
**Pull request by zhongwencool**
delete emqx_trace_api, replace LOG by SLOG

### CODE REVIEW

File: apps/emqx_management/test/emqx_mgmt_trace_api_SUITE.erl [hunk 1]
```diff
@@ -112,6 +114,30 @@ t_http_test(_Config) ->
...
 
+t_download_log(_Config) ->
+    ClientId = <<"client-test-download">>,
+    Now = erlang:system_time(second),
+    Start = to_rfc3339(Now),
+    Name = <<"test_client_id">>,
+    load(),
+    ok = emqx_trace:create([{<<"name">>, Name},
+        {<<"type">>, clientid}, {<<"clientid">>, ClientId}, {<<"start_at">>, Start}]),
+    ct:sleep(50),
```
Comment (zmstone) [inline apps/emqx_management/test/emqx_mgmt_trace_api_SUITE.erl:125]: Nit: Use a snabbkaffe trace and wait for it

**Review by zmstone — APPROVED** [review]

File: apps/emqx_management/test/emqx_mgmt_trace_api_SUITE.erl (hunk 1 above)
Comment (zhongwencool) [reply apps/emqx_management/test/emqx_mgmt_trace_api_SUITE.erl]: done

**Comment by zmstone** [comment]
> delete emqx_trace_api, replace LOG by SLOG

Nit: elaborate why it is deleted?

**Review by zmstone — APPROVED** [review]

**Review by terry-xiaoyu — APPROVED** [review]

**Comment by zhongwencool** [comment]
> > delete emqx_trace_api, replace LOG by SLOG
> 
> Nit: elaborate why it is deleted?

The emqx_trace_api is replace by emqx_mgmt_api_trace. 
this module is port from 4.4, 4.4's api schema don't use hocon schema, 
but we write api schema and api in the same module(emqx_mgmt_api_trace), emqx_trace_api  is no longer needed

**Pull request merged**